langroid/vector_store/qdrantdb.py

```diff
@@ -71,6 +80,15 @@ def __init__(self, config: QdrantDBConfig = QdrantDBConfig()):
         emb_model = EmbeddingModel.create(config.embedding)
         self.embedding_fn: EmbeddingFunction = emb_model.embedding_fn()
         self.embedding_dim = emb_model.embedding_dims
+        if self.config.use_sparse_embeddings:
+            from transformers import AutoModelForMaskedLM, AutoTokenizer
```

Comment:
Please add a try/except block around this, with a warning saying user needs to install langroid with the "transformers" extra. See here for an example -- 

https://github.com/langroid/langroid/blob/main/langroid/language_models/prompt_formatter/hf_formatter.py#L31

Reply:
@pchalasani looks like the collection created does not have sparse vector.
check config of the collection created, it should have both dense-vector and sparse-vector in the config.

Reply:
Not exactly same , but a similar one that suits the context of that import.